website_v2/gcloud-powershell-website-v2/www/static/app/main_controller.js

```diff
@@ -46,14 +46,13 @@ app.controller('MainController',
       this.order = ['synopsis', 'syntax', 'description', 'parameters',
           'examples', 'inputs', 'outputs'];
 
-      /* Gets the parameters for the productInfo passed in */
-      this.getParams = function(productInfo) {
-        var cmdlet = productInfo[$routeParams.product][$routeParams.cmdlet];
-        return cmdlet.parameters;
-      };
-      
       /* Tells us if the object passed in is empty */
       this.isEmpty = function(linkObject) {
-        return (Object.keys(linkObject) === 0);
-      }
+        return (Object.keys(linkObject).length === 0);
+      };
+
+      /* Gets the relevant cmdlet property from the productInfo */
+      this.getProperty = function(productInfo,  property) {
+        return productInfo[$routeParams.product][$routeParams.cmdlet][property];
```

Comment:
Is this ever going to be called with something other than `mainCtrl.productInfo`? If not, consider just having this function take `property` and return `$scope.productInfo[...`.